dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/sms/scheduling/SendScheduledMessageJob.java

```diff
@@ -66,19 +64,15 @@ public JobType getJobType() {
   @Override
   public void execute(JobConfiguration config, JobProgress progress) {
     progress.startingProcess("Starting to send messages in outbound");
```

Comment:
this reads a bit funny `Starting to send messages in outbound`
should it be `Starting to send messages outbound` or `Starting to send messages in outbound {something}`?

Reply:
I rephrased it